docs/user-guides/finetuner.md

```diff
@@ -0,0 +1,144 @@
+# Fine-tune Models
```

Comment:
Can we have a colab notebook for this purpose?

Reply:
i could, but have to add some loop to wait for finetuner.run

Reply:
In practice, the user should also add a loop to make sure the job is done. This notebook can provide more practical tips then.